### PR TITLE
Enable arm64 zip and runtime pack nuget generation

### PR DESCRIPTION
**Pull request by NikolaMilosavljevic**
Fixes https://github.com/dotnet/windowsdesktop/issues/530

### CODE REVIEW

**Comment by AdamYoblick** [comment]
I'm not familiar enough with this repo nor the crossgen process to comment on this PR. It looks good to me, but Davis would know much better. 😄 

Where can I check that the artifacts are, in fact, native arm64 binaries? The output of this build only seems to have logs...do we have to wait for a non-pr build to get real artifacts?

**Comment by NikolaMilosavljevic** [comment]
> Where can I check that the artifacts are, in fact, native arm64 binaries? The output of this build only seems to have logs...do we have to wait for a non-pr build to get real artifacts?

You can check my test build: https://dev.azure.com/dnceng/internal/_build/results?buildId=615784&view=results

All packages will be in artifact called "PreparedArtifacts"

File: pkg/windowsdesktop/sfx/Microsoft.WindowsDesktop.App.Bundle.bundleproj [hunk 1]
```diff
@@ -17,24 +17,29 @@
...
-  <ItemGroup>
-    <InsertionPackageRID Include="@(RestoreBuildRID)" Arch="$([System.String]::new('%(Identity)').Replace('win-', ''))" />
+  <Target Name="CollectVSInsertionPackageDownloads"
+          BeforeTargets="CollectPackageReferences">
 
-    <NETCoreAppInstallerMsiRID Include="@(InsertionPackageRID)" Name="runtime" InsertionName="SharedFramework" />
-    <NETCoreAppInstallerMsiRID Include="@(InsertionPackageRID)" Name="hostfxr" InsertionName="HostFXR" />
-    <NETCoreAppInstallerMsiRID Include="@(InsertionPackageRID)" Name="host" InsertionName="SharedHost" />
+    <ItemGroup>
+      <InsertionPackageRID Include="@(RestoreBuildRID)" Arch="$([System.String]::new('%(Identity)').Replace('win-', ''))" Condition="'%(RestoreBuildRID.RuntimeMsiExists)' == 'true'"/>
```
Comment (dagood) [inline pkg/windowsdesktop/sfx/Microsoft.WindowsDesktop.App.Bundle.bundleproj:24]: I would chop this for readability:

```suggestion
      <InsertionPackageRID
        Include="@(RestoreBuildRID)"
        Arch="$([System.String]::new('%(Identity)').Replace('win-', ''))"
        Condition="'%(RestoreBuildRID.RuntimeMsiExists)' == 'true'" />
```

Comment (NikolaMilosavljevic) [reply pkg/windowsdesktop/sfx/Microsoft.WindowsDesktop.App.Bundle.bundleproj:24]: yeah, I was considering that, but before wrapping the group in target it wasn't as bad. I'll update it

**Comment by hmartinez82** [comment]
@NikolaMilosavljevic I saw you updated the build code. But I just tried to run
D:\Dev\Github\windowsdesktop [master ≡]> .\build.cmd -c Release -platform arm64 and it gave me x64 binaries.

**Comment by NikolaMilosavljevic** [comment]
> @NikolaMilosavljevic I saw you updated the build code. But I just tried to run
> `D:\Dev\Github\windowsdesktop [master ≡]> .\build.cmd -c Release -platform arm64` and it gave me x64 binaries.
> 
> How can one build the arm64 locally?

build /p:TargetArchitecture=arm64

TargetArchitecture is used in official build to specify correct architecture, see https://github.com/dotnet/windowsdesktop/blob/caaf3d8989b33e7b894d84514046e7ffa3759dd5/azure-pipelines.yml#L67

**Comment by hmartinez82** [comment]
@NikolaMilosavljevic what the steps you're using to build locally in your machine. Not in Azure Devops?

**Pull request merged**